test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -230,12 +232,117 @@ public void SdkLookup_Must_Look_For_Available_Versions_Before_Looking_Into_Anoth
                 .Should()
                 .Pass()
                 .And
-                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0-global-dummy"));
+                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0-global-dummy", _dotnetSdkDllMessageTerminator));
 
             // Remove dummy folders from user dir
             DeleteAvailableSdkVersions(_userSdkBaseDir, "9999.0.0", "9999.0.0-dummy");
         }
 
+        [Fact]
+        public void SdkLookup_Must_Pick_The_Latest_Semantical_Version()
```

Comment:
I wouldn't call it latest, I would call it highest.

Btw, good to see this test here.